internal/provider/resource_application.go

```diff
@@ -122,6 +129,92 @@ func (r *applicationResource) Configure(ctx context.Context, req resource.Config
 	r.subCtx = tflog.NewSubsystem(ctx, LogResourceApplication)
 }
 
+func storageSetRequiresReplace(ctx context.Context, req planmodifier.SetRequest, resp *setplanmodifier.RequiresReplaceIfFuncResponse) {
```

Comment:
This belongs in a new file, `modifer_storageset.go`, as the validators are.

Reply:
Not sure that I know where to put this file.